Purchases/Purchasing/TrialOrIntroPriceEligibilityChecker.swift

```diff
@@ -92,6 +93,23 @@ class TrialOrIntroPriceEligibilityChecker {
         return introDict
     }
 
+    @available(iOS 11.2, macOS 10.13.2, macCatalyst 13.0, tvOS 11.2, watchOS 6.2, *)
+    func productsWithIntroOffers(productIdentifiers: [String], completion: @escaping ReceiveIntroEligibilityBlock) {
```

Comment:
this method could be `private`, right?

Reply:
I had it private, originally, but then I wanted to mock it and so I switched it back. I ended up not mocking it and forgetting to move it back to the private extension 🤦‍♂️

Reply:
✅